bears/vcs/git/GitCommitBear.py

```diff
@@ -113,7 +113,7 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
             shortlog,
```

Comment:
ahh...
please don't implement that in a commit before, and then you move it. Make it right from the beginning, implement it directly in `self.check_body` :+1: